cmd/gaia/app/util.go

```diff
@@ -0,0 +1,22 @@
+package app
+
+import (
+	"io"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	bam "github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	dbm "github.com/tendermint/tendermint/libs/db"
+)
+
+// used for debugging by gaia/cmd/gaiadebug
+// NOTE to not use this function with non-test code
```

Comment:
Then can we put it in a `_test.go` file?

Reply:
not really because you can't import files from `_test.go` files. I could however rename to `test_utils.go` could be better?

note this also needs to remain in `app/` due to accessing the private fields

Reply:
ah tragic - I like `test_utils.go`

Reply:
yeah I've added to test_utils